chord_metadata_service/mcode/models.py

```diff
@@ -1,342 +0,0 @@
-from django.utils import timezone
-from django.db import models
-from django.db.models import JSONField
-from django.contrib.postgres.fields import ArrayField
-from chord_metadata_service.restapi.models import IndexableMixin
-from chord_metadata_service.phenopackets.models import Gene
-from chord_metadata_service.patients.models import Individual
-from chord_metadata_service.restapi.description_utils import rec_help
-import chord_metadata_service.mcode.descriptions as d
-from chord_metadata_service.restapi.validators import ontology_validator, ontology_list_validator
-from .validators import (
```

Comment:
can we also remove this validators file that mcode.models was importing from?

Reply:
actually, there are many other files (schemas, serializers etc) not removed yet in this PR...

Reply:
Updated, only left the files/folders required for migrations